README.md

```diff
@@ -49,7 +54,9 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+town[:residents].delete_at(1)
+town[:castle][:residents] = [town[:castle][:residents]]
+town[:castle][:residents].push("Belle")
 ```
```

Comment:
This would be a very clever solution *if* the question had asked you to add Belle to the castle's residents, rather that its guests. :wink: